packages/blocks/src/store/selectors.js

```diff
@@ -57,6 +57,18 @@ export function getBlockStyles( state, name ) {
 	return state.blockStyles[ name ];
 }
 
+/**
+ * Returns block patterns by block name.
+ *
+ * @param {Object} state      Data state.
+ * @param {string} blockName  Block type name.
+ *
+ * @return {?(WPBlockPattern[])} Block patterns.
```

Comment:
Recent focus on types has me wondering: Does `?` mean the return value is `null`-able? If so, when is it `null`? In fact, it can return `undefined`, but in that case do we need to express it differently?

```suggestion
 * @return {(WPBlockPattern[]|void)} Block patterns.
```

Reply:
I'd love to learn what's the proper way of signaling in JSDoc that something can be an array of items or undefined. I will fly with your suggestion until I learn more.